src/app/lib/utilities/passport/index.js

```diff
@@ -0,0 +1,38 @@
+import pathOr from 'ramda/src/pathOr';
+
+export const getPassportHome = pageData => {
+  const homeServiceUrl = pathOr(
+    null,
+    ['metadata', 'passport', 'home'],
+    pageData,
+  );
+  const passportHomeOverride = pathOr(
+    null,
+    ['metadata', 'passportHome'],
+    pageData,
+  );
+  return (
+    passportHomeOverride ||
```

Comment:
Are there any tests for `passportHome` taking precedence over `passport.home`?

Reply:
Sure! https://github.com/bbc/simorgh/pull/3773/files#diff-740a11081120190bbc20bb73a8ee1f08R46-R48